EPCPortalWeb/Models/ReportDataModel.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EPCPortalWeb.Models
+{
+    public class ReportDataModel
+    {
+        // class to structure data returned from the API into usable items for properties and methods
+
+        public double CurrHeatingCost { get; set; }
```

Comment:
Might be more readable to use full word "Current" in the property name, which is consistent with your using the full word "Potential" below.